install/settings.json

```diff
@@ -148,5 +148,65 @@
     "value": {
       "default": "<p>Note: We don't have data for the country selected, so we could not adjust the GNI per capita accordingly, but you can still sign up! Please contact us for a custom quote.</p>"
     }
+  },
+  {
+    "group": {
+      "name": "plugin:consortial_billing"
```

Comment:
I understand why you'd want to use the plugin naming scheme for the group, but bear in mind these will not be seen as email templates elsewhere in Janeway (e.g. email template manager). setting the group as `"email"` is the only way to achieve that at the moment.

If you wanted this behaviour, every email setting will also need a `"email_subject"` group member as a counterpart.